renderer.go

```diff
@@ -112,7 +112,7 @@ func (r *Renderer) resetPrompt(lines int) {
 	terminal.EraseLine(r.stdio.Out, terminal.ERASE_LINE_ALL)
 	// clean up what we left behind last time
 	for i := 0; i < lines; i++ {
-		cursor.PreviousLine(1)
```

Comment:
If i'm following what you found, it seems like `cursor.PreviousLine` method should be removed entirely since it isn't uniformly supported. I would hate for me to forget this was a problem and bring up the issue in another spot. 

Thoughts?

Reply:
We could either remove it entirely or leave a comment saying it is deprecated since it is not supported by all terminals. Deprecating this method would avoid having to move the module to v2, because removing the method would be a breaking change.

Reply:
@AlecAivazis 
I made a bit more testing to make sure other inputs still work properly, especially the ones using `PreviousLine()`. All inputs appear to work properly for me.

`PreviousLine()` still appears to be required. It is used in many other places, so I wouldn't remove it nor mark it as deprecated.

Reply:
Sorry if i'm missing something obvious, but I thought the whole source of this problem is that Konsole doesn't support `PreviousLine`. How does it work in some situations but not in others?

Reply:
Honestly I'm not sure. But if that works correctly like it is now, I would avoid removing things that may break the lib on other terminals.

Reply:
Ah I think I figured it out! If I understood correctly `PreviousLine()` set the cursor at the **end** of the previous line. `resetPrompt()` was erasing starting at the cursor position towards the end of the line.